backend/src/main/scala/cromwell/backend/BackendJobCachingActor.scala

```diff
@@ -0,0 +1,47 @@
+package cromwell.backend
+
+import akka.actor.{Actor, ActorLogging}
+import akka.event.LoggingReceive
+import cromwell.backend.BackendJobCachingActor._
+import cromwell.backend.BackendJobExecutionActor.{BackendJobExecutionResponse, FailedNonRetryableResponse}
+import cromwell.backend.BackendLifecycleActor._
+import cromwell.core.JobOutputs
+
+import scala.concurrent.Future
+
+//1. Unclear if Abort needs to be a part of Caching actor? What should happen when an abort command hits during caching and not execution?
```

Comment:
I would think a caching actor should stop copying stuff if it gets an Abort, but yeah we should discuss this as a group